pkg/controller/appdefinition/status.go

```diff
@@ -405,7 +405,12 @@ func endpoints(req router.Request, cfg *apiv1.Config, app *v1.AppInstance) (stri
 			}
 
 			if endpoint.Pending {
-				buf.WriteString("<Pending Ingress>")
+				switch endpoint.Protocol {
+				case "tcp":
```

Comment:
Technically I believe "UDP" is also supported.  I would probably just check for http and set "Pending Ingress" and for everything else "Pending Load Balancer"